parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -300,16 +308,15 @@ write_files:
     runtime-endpoint: unix:///run/containerd/containerd.sock
     #EOF
 
-{{if IsKubenet }}
 - path: /etc/sysctl.d/11-containerd.conf
   permissions: "0644"
   owner: root
   content: |
     net.ipv4.ip_forward = 1
```

Comment:
actually I'm adding a new file /etc/sysctl.d/999-sysctl-aks.conf. Consider moving these settings to it

Reply:
sure will put add this to the new systctl.conf once your PR gets merged. leaving this as is for now.